contracts/orbital-core/schema/raw/execute.json

```diff
@@ -0,0 +1,228 @@
+{
```

Comment:
Let's add all the raw folders to .gitignore imo like we are doing in the other repos (it's pretty standard in general). Or actually you can make the script remove them after generating them. We already confirmed with Lena that she doesn't need them.

Reply:
sounds good!